docs/content/en/docs/concepts/cluster-topologies.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Cluster topologies"
+linkTitle: "Cluster topologies"
+weight: 20
+description: >
+  Explanation of standalone vs. management/workload cluster topologies
+---
+
+For trying out EKS Anywhere or for times when a single cluster is needed, it is fine to create a _standalone cluster_ and run your workloads on it.
+However, if you plan to create multiple clusters for running Kubernetes workloads, we recommend you create a _management cluster_.
+Then use that management cluster to manage a set of workload clusters.
+
+This document describes those two different EKS Anywhere cluster topologies.
+
+## What is an EKS Anywhere management cluster?
+An EKS Anywhere management cluster is a long-lived, on-premises Kubernetes cluster that can create and manage a fleet of EKS Anywhere workload clusters.
+The workload clusters are where customers run their applications.
+The management cluster can only be created and managed by the Amazon CLI `eksctl`.
+
+The management cluster runs on customer hardware on-premises and it does not require any connectivity back to AWS to function.
+Customers are responsible for operating the management cluster including (but not limited to) patching, upgrading, scaling, and monitoring the cluster control plane and data plane.
+ 
+## What’s the difference between a management cluster and a standalone cluster?
+From a technical point of view, they are the same.
+Regardless of which deployment topology a customer chooses, you always start by creating a singleton, standalone cluster that’s capable of managing itself.
```

Comment:
"Regardless of which deployment topology you choose, you always start by creating..."

Reply:
Done.